src/python/WMComponent/DBS3Buffer/MySQL/DBSBufferFiles/AddRunLumi.py

```diff
@@ -36,7 +36,8 @@ def getBinds(self, filename=None, runs=None):
                 for lumi in run:
```

Comment:
New code doesn't have run.lumis. __iter__ is redefined so it should be ok with "for lumi in run"
https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/DataStructs/Run.py#L76